### PR TITLE
v1.1 React Native

### PR DESCRIPTION
**Pull request by laves**

### CODE REVIEW

File: binding/react-native/README.md [hunk 1]
```diff
@@ -37,7 +34,7 @@ or
 npm i @picovoice/cheetah-react-native --save
 ```
 
-Link the iOS package
+Link the iOS packageL
```
Comment (ksyeo1010) [inline binding/react-native/README.md:37]:
```suggestion
Link the iOS package:
```

**Review by ksyeo1010 — CHANGES_REQUESTED** [review]

**Comment by ksyeo1010** [comment]
All good but in the demo theres some times that the transcription flickers and its not shown

**Comment by laves** [comment]
> All good but in the demo theres some times that the transcription flickers and its not shown

Okay, I think I fixed it. I think there's a race condition in the app with reading the current state of the transcript and updating it. Not storing the state in a variable for both outcomes seemed to help.

**Review by ksyeo1010 — APPROVED** [review]

**Comment by laves** [comment]
packages have been released, so you can merge

**Pull request merged**